test/test_sbol2_sbol3_direct.py

```diff
@@ -299,6 +299,9 @@ def handle_2to3_conversion(self, test_filename: str, comparison_filename: str):
     def test_implementation_conversion(self):
         self.handle_2to3_conversion('sbol_3to2_implementation_compliant.xml', 'sbol_3to2_implementation_compliant.nt')
 
+    def test_moduledefinition_conversion(self):
+        self.handle_2to3_conversion('sbol_3to2_moduledefinition.xml', 'sbol_3to2_moduledefinition.nt')
```

Comment:
Let us also test with `handle_3to2_conversion`, in the other half of the file.

Reply:
Added in 883d7268f82323c4e4f4b8d98e386f87e7bebf8a